charts/kserve/values.yaml

```diff
@@ -413,7 +413,7 @@ knativeServingIstio:
   enabled: true
   helmChart:
     chart: knative-serving-istio
-    version: 1.4.0
+    version: 1.4.1
```

Comment:
@mbruner  shall i update this to the latest major version too?

Reply:
do you mean `1.7.x`? I think yes, should be updated since it was already tested together